api/raft.proto

```diff
@@ -16,6 +16,13 @@ service Raft {
 		option (docker.protobuf.plugin.tls_authorization) = { roles: "swarm-manager" };
 	};
 
+	// RaftMessageStream returns a gRPC stream that can be used to stream raft messages
+	// to be processed on a raft member.
+	// It is called from the RaftMember willing to send a message to its destination ('To' field)
+	rpc RaftMessageStream(stream ProcessRaftMessageRequest) returns (ProcessRaftMessageResponse) {
```

Comment:
This needs a response type wrapper.

Reply:
This seems confusing. The doc says that it returns a stream, but it only seems to take a stream as input. Who is the caller and callee in this scenario?